.circleci/config.yml

```diff
@@ -129,6 +129,27 @@ jobs:
               "${ARNOLD_IMAGE}" \
               ansible --version
 
+  test-bootstrap:
+    machine: true
+    working_directory: ~/fun
+    steps:
+      - checkout
+      - *attach_workspace
+      - *docker_load
+      - *ci_env
+      - run:
+          name: Test deploy in OpenShift
+          command: |
+            sudo bash -c "echo '{\"insecure-registries\": [\"172.30.0.0/16\"]}' > /etc/docker/daemon.json"
+            sudo service docker restart
+            wget https://github.com/openshift/origin/releases/download/v3.9.0/openshift-origin-client-tools-v3.9.0-191fece-linux-64bit.tar.gz
+            tar -xf openshift*.tar.gz --strip-components=1
```

Comment:
Why not using `tar xzf openshift*.tar.gz`? In a temporary directory and make an install (copy `oc`) in `$HOME/bin`?